apps/src/lib/util/firehose.js

```diff
@@ -118,6 +119,8 @@ class FirehoseClient {
   getLocale() {
     if (window.appOptions) {
       return window.appOptions.locale;
+    } else {
+      return cookies.get('language_') || 'en-US';
```

Comment:
Please try to use this function if possible: https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/util/currentLocale.ts

Reply:
Awesome! Thanks, I thought we had some function, but I could not find it.